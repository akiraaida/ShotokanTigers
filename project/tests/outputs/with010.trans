10 Anon Guy             00000 00000.00 S 
01 Anon Guy             12349 00010.00   
01 Anon Guy             12349 00000.05   
01 Anon Guy             12350 00495.00   
01 Anon Guy             12350 00000.10   
01 Anon Guy             12349 00490.00   
01 Anon Guy             12349 00000.05   
01 Anon Guy             12350 00005.00   
01 Anon Guy             12350 00000.10   
00                      00000 00000.00   
