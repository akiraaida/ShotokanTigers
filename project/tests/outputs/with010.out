Please enter a command: 
Please enter your session type: 
Please enter a login name: 
Please enter a command: 
Please enter the user's account number: 
Please enter amount to withdraw: 
The withdrawal transaction has completed.
Available Balance (12349):
1090.91
Daily Withdrawals Remaining (12349):
490.00
Please enter a command: 
Please enter the user's account number: 
Please enter amount to withdraw: 
The withdrawal transaction has completed.
Available Balance (12350):
105.70
Daily Withdrawals Remaining (12350):
5.00
Please enter a command: 
Please enter the user's account number: 
Please enter amount to withdraw: 
The withdrawal transaction has completed.
Available Balance (12349):
600.86
Daily Withdrawals Remaining (12349):
0.00
Please enter a command: 
Please enter the user's account number: 
Please enter amount to withdraw: 
The withdrawal transaction has completed.
Available Balance (12350):
100.60
Daily Withdrawals Remaining (12350):
0.00
Please enter a command: 
